### DISCUSSION TITLE
streamMetabolizer and Arctic streams

### DISCUSSION THREAD

**Discussion by NicoleGotkowski** [USGS-R/streamMetabolizer#413] [Q&A]
Hi all,

I am relatively new to streamMetabolizer, and I am trying figure out how to use streamMetabolizer to measure GPP, ER and K600 in Arctic streams that experience days with 0 periods of PAR=0. I had a colleague that once asked this question at an SFS conference and he was told it would not work due to the equations being used.

Many (~10) days do experience periods of PAR=0. As well, I have noticed distinct daily changes in oxygen. On top of this, many of my streams end up producing results that make sense. 

I was wondering if it is true that the equations in streamMetabolizer are unusable when some days do not experience PAR=0.

Thanks in advance!

Nicole

**Comment by robohall** [comment]
Nicole,

It should work.  The model responds to GPP as a linear function of light and even though it does not get perfectly dark, it is a lot darker at 2 am than it is at 2 pm.  So give it a try using the light data you have at your site.  That light data should be an unobstructed view of the sky.

Not sure who said it won’t work  Those are strong words without trying first.  It may be that because it does not get perfectly dark you do not get a perfect estimate of ER.  But an acknowledged imperfect estimate is far better than no estimate at all.  A way to convince yourself and reviewers that this method can work is to simulate data (using streamMetabolizer), given inputs of GPP, ER, K600, light, process error and observation error (get these error estimates from runs with real data).  That is, make a fake oxygen dataset that is say a month long.  .  Then use sM to estimate the parameters as if it were a real dataset and see the degree you which you get back estimates of ER and GPP relative to ER and GPP you used to generate the fake data.


The biggest reason this method won’t work is if you cannot parse out ER and K, so after you analyze a lot of days, plot ER as a function of K600.  If they covary, you have equifinality.  If not ok.

Bob



On Jan 27, 2022, at 11:12 AM, NicoleGotkowski ***@***.******@***.***>> wrote:



Hi all,

I am relatively new to streamMetabolizer, and I am trying figure out how to use streamMetabolizer to measure GPP, ER and K600 in Arctic streams that experience days with 0 periods of PAR=0. I had a colleague that once asked this question at an SFS conference and he was told it would not work due to the equations being used.

Many (~10) days do experience periods of PAR=0. As well, I have noticed distinct daily changes in oxygen. On top of this, many of my streams end up producing results that make sense.

I was wondering if it is true that the equations in streamMetabolizer are unusable when some days do not experience PAR=0.

Thanks in advance!

Nicole

—
Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Fdiscussions%2F413&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfe907ca9849c40abea0a08d9e1c08bc4%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789039373164402%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=dctKF671LlcFcfmObsu1lCTwX0l3AYuJZcpQnGaj4iI%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC4CU5QYLLT3CW6K62CTALTUYGDHXANCNFSM5M6VTDSQ&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfe907ca9849c40abea0a08d9e1c08bc4%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789039373164402%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=VwBfu%2FfILFPf4quzd1poRa7CnN%2F3HnIfw0KE4ACsrt8%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfe907ca9849c40abea0a08d9e1c08bc4%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789039373164402%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=db%2FG%2BGEiYlWkLiNeToZ1IMVsCuYusgyizp69%2FUGnb9w%3D&reserved=0> or Android<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfe907ca9849c40abea0a08d9e1c08bc4%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789039373164402%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=kUXuDxRx4r5m8CV4repPsQd2KnEUAObOOalkRKAZSF8%3D&reserved=0>.

**Reply by NicoleGotkowski** [reply]
Thank you for your timely response!

This is what I had suspected when I saw how GPP is a linear function of light when reading Appling et al., 2018. 

I did something similar to what you have suggested. I created several data sets with PAR thresholds, i.e., I created artificial data sets, taken from my own data, where when PAR<15, it would be set so that it equaled zero. I had suspected there would be no change, and I was correct. I went all the way from PAR 15, PAR 20, PAR 50, PAR 100, PAR 1000, for 16 sites (some sites were unusable due to weak signals). To ensure this held true, I also cut off my data sets to only use dates where PAR=0 for 12 time steps, so the beginning of August, and again, I saw no significant differences. However, some rhat values changed from an unacceptable value to an acceptable value.

Would you argue this is adequate to justify to reviewers?

Thanks again,

Nicole.

**Comment by robohall** [comment]
Without seeing the results myself, I think this will be enough.  My feeling is that reviewers don’t argue if you think about the problem before they do and then use data to show that complete darkness is unnecessary to get reasonable estimates of metabolism.

Bob

On Jan 27, 2022, at 12:37 PM, Nicole Gotkowski ***@***.******@***.***>> wrote:



Thank you for your timely response!

This is what I had suspected when I saw how GPP is a linear function of light when reading Appling et al., 2018.

I did something similar to what you have suggested. I created several data sets with PAR thresholds, i.e., I created artificial data sets, taken from my own data, where when PAR<15, it would be set so that it equaled zero. I had suspected there would be no change, and I was correct. I went all the way from PAR 15, PAR 20, PAR 50, PAR 100, PAR 1000, for 16 sites (some sites were unusable due to weak signals). To ensure this held true, I also cut off my data sets to only use dates where PAR=0 for 12 time steps, so the beginning of August, and again, I saw no significant differences. However, some rhat values changed from an unacceptable value to an acceptable value.

Would you argue this is adequate to justify to reviewers?

Thanks again,

Nicole.

—
Reply to this email directly, view it on GitHub<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FUSGS-R%2FstreamMetabolizer%2Fdiscussions%2F413%23discussioncomment-2061619&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfc40401593f146f18d2108d9e1cc8756%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789090838197523%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6TzUwwr5vCoOrgUYdpHlLneTXEaMoESa2XVx7EF4QDw%3D&reserved=0>, or unsubscribe<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAC4CU5RTRAAREWPRNDZZUHDUYGNJNANCNFSM5M6VTDSQ&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfc40401593f146f18d2108d9e1cc8756%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789090838197523%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=9jHdeuwDhzqoC1cgeGRoE145sSIJS0jji0VR1hbZCqc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfc40401593f146f18d2108d9e1cc8756%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789090838197523%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=3G76gzfsXgM00x5pOIPLWJr1dZMYUOZ%2F%2FyexKj0HkRg%3D&reserved=0> or Android<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cbob.hall%40flbs.umt.edu%7Cfc40401593f146f18d2108d9e1cc8756%7C68407ce503da49ffaf0a724be0d37c9d%7C0%7C0%7C637789090838197523%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=7M0rOoxTsN8z%2FJGjtpZbxYZbZOVn14QJEbg8enFVRO4%3D&reserved=0>.

**Reply by NicoleGotkowski** [reply]
Great, thanks for your help!
